audit_history/utils.py

```diff
@@ -17,7 +17,9 @@ def json_formatter(o):
         if o.tzinfo:
             return o.strftime('%H:%M:%S%z')
         return o.strftime("%H:%M:%S")
+    if isinstance(o, datetime.tzinfo):
+        return o.zone
     if isinstance(o, set):
         return list(o)
 
-    return o
+    raise TypeError('Unserializable object {} of type {}'.format(o, type(o)))
```

Comment:
Am I missing something here, or does this mean that strings, ints, bools, etc. can no longer be JSON formatted?

Reply:
It's used for situation where we need additional instructions for json convertation. object_hook is an optional function that will be called with the result of any object literal decoded (a dict). The return value of object_hook will be used instead of the dict. This feature can be used to implement custom decoders (e.g. JSON-RPC class hinting)(https://docs.python.org/2/library/json.html). I add this for recursion avoiding if we met unserializable object (like model without to_json() method)